node/src/cli.rs

```diff
@@ -153,57 +148,6 @@ impl Cli {
                 let validator_config = Self::init(&config, config_ext)?;
                 info!(version = %crate::VERSION_STRING.as_str(), "node starting up");
 
-                let pid_file_outcome = {
-                    // Determine storage directory to store PidFile in.
-                    let storage_config = validator_config.map_ref(|cfg| cfg.storage.clone());
-                    let root = storage_config.with_dir(storage_config.value().path.clone());
-
-                    // Create directory if it does not exist, similar to how the storage component
-                    // would do it.
-                    if !root.exists() {
-                        fs::create_dir_all(&root).context("create storage directory")?;
-                    }
-
-                    PidFile::acquire(root.join("initializer.pid"))
-                };
-
-                // Note: Do not change `_pid_file` to `_`, or it will be dropped prematurely.
-                // Instantiating `pid_file` guarantees that it will be dropped _after_ any reactor,
-                // which is what we want.
-                let (_pid_file, pid_outcome_check_integrity) = match pid_file_outcome {
-                    PidFileOutcome::AnotherNodeRunning(_) => {
-                        anyhow::bail!("another node instance is running (pid_file is locked)");
```

Comment:
Maybe we should keep the pid file, and still bail or print the warning, even without the integrity check?

Reply:
The removal of pid file was briefly discussed before being put into effect. Surely, I'm open to further changes in this regard.

Let's wait for the feedback from other reviewers.

Reply:
Sure, just a thought. If it was discussed I'm fine with it. :+1:

Reply:
I'd support leaving it in order to detect that another node instance is running, but I don't feel strongly about it.

Reply:
We can bring the pidfile back if we do have a need for it in the future.  For now, I believe we should just remove it.